WordPress/src/androidTest/java/org/wordpress/android/e2e/ReaderTests.java

```diff
@@ -26,8 +28,9 @@ public void setUp() {
     String mCoachingPostTitle = "Let's check out the coaching team!";
     String mCompetitionPostTitle = "Let's focus on the competition.";
 
-//    @Test
-    public void navigateThroughPosts() {
+    @Ignore("Ignored temporarily. This sometimes fail on CI while running with whole test suite.")
+    @Test
+    public void e2eNavigateThroughPosts() {
```

Comment:
Two tests in this file were disabled via `//@Test`, not via `@Ignore`, which made them not show up on FTL as skipped. So, it turns out, we have 9 disabled tests, not 7.